persistence-modules/spring-boot-persistence-5/src/main/java/com/baeldung/transactional/TestOrder.java

```diff
@@ -0,0 +1,21 @@
+package com.baeldung.transactional;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+import lombok.Getter;
```

Comment:
We have a guideline to not use Lombok in articles that aren't specifically demonstrating Lombok features